frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -414,6 +465,42 @@ export class Query extends Component {
         ) : (
           <React.Fragment>
             <NavBar />
+            <Modal
+              id="fuzzy-warning"
+              open={!this.state.showMismatch}
+              onClose={()=> this.setState({showMismatch:true,loading:false})}
+              size='tiny'
+              >
+              <Modal.Header>Warning!
+                <Icon
+                name="times"
+                className="times-icon"
+                onClick={()=> this.setState({showMismatch:true,loading:false})}
+              />
+              </Modal.Header>
+              <Modal.Content>
+                <h4>The usernames you have provided does not seem to belong to the same user.
```

Comment:
Enclose this in a <p> tag instead to reduce the boldness.

Reply:
Ok